### DISCUSSION TITLE
Proposed Transactions meeting agenda for 2023-02-22

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-transactions#6] [Meetings]
## Meeting Details

- **Date/Time:** February 22th, 2023 @ Wednesday 1:00 PM ET / 18:00 UTC / 7:00 PM CET
- **Location:** Link should be posted in the `#Transactions` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** TBA
- **Note Taker** Person

The [FIG Transactions Meetings](https://github.com/Open-MV/fig-transactions/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda
Breaking down and prioritizing database entries
Discussing which items to add to security and privacy agenda
Update on charter items added as database elements.

**Discuss agenda from proposed topics**

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by will-a123** [comment]
I'm not available for a call today, but have made myself a ticket to research the current top 5 micropayments specialists and put together a brief summary of each -- assuming that we still feel that the OMF's eventual uber-protocol should have some build in support for the concept of micropayments, regardless of whether or not it gets used (sort of like HTTP 402)

**Reply by OBWANDO** [reply]
I can see micropayments being a big part of the metaverse, its proven itself over many years in the games industry. People like bite sized morsels of fun.
